### DISCUSSION TITLE
The sign in option won't disappear after signing in VSCODE

### DISCUSSION THREAD

**Discussion by CreepyJ** [microsoft/vscode-discussions#946] [Extension Development QnA]
I'm developing a Visual Studio Code extension and trying to create my own Authentication Provider. 
I've noticed that the 'sign in' option doesn't disappear even after I've already signed in. 
I have to reopen Visual Studio Code to get rid of it. 
Does anyone know the reason behind this and how to fix it?
![image](https://github.com/microsoft/vscode-discussions/assets/39795353/6605fcb0-c966-4231-950f-3bf243bde091)

**Comment by olguzzar** [comment]
Hi @CreepyJ, it's been a while since there's been activity on this discussion. Are you still having this issue?

**Comment by isidorn** [comment]
Closing since more details are needed. If you can repro please file a new issue in the vscode repo
